source/standards/how-to-do-penetration-tests.html.md.erb

```diff
@@ -1,49 +1,73 @@
 ---
 title: How to do penetration tests
-last_reviewed_on: 2019-03-26
+last_reviewed_on: 2019-11-13
 review_in: 6 months
 ---
-
+​
 # <%= current_page.data.title %>
-
-You should aim to run [penetration tests](https://www.gov.uk/service-manual/technology/vulnerability-and-penetration-testing) on your service every 6 months. You must work with the [GDS Information Assurance (IA) team](https://sites.google.com/a/digital.cabinet-office.gov.uk/gds/operations/information-assurance) to agree when you will test and to procure external tests.
-
-Tests should be carried out by an approved third party through the [National Cyber Security Centre (NCSC) CHECK scheme](https://www.ncsc.gov.uk/articles/check-fundamental-principles) or internally by a member of the GDS Cyber Security Team, depending on your need.
-
+​
+You should aim to run [penetration tests](https://www.gov.uk/service-manual/technology/vulnerability-and-penetration-testing) on your service at least every 12 months. You must work with the [GDS Information Assurance (IA) team](https://sites.google.com/a/digital.cabinet-office.gov.uk/gds/operations/information-assurance) to agree when you will test and to procure external tests.
+​
+An approved third party should carry out tests through the [National Cyber Security Centre (NCSC) CHECK scheme](https://www.ncsc.gov.uk/articles/check-fundamental-principles) or a member of the GDS Cyber Security Team can carry them out internally, depending on your requirements.
+​
 You may need to schedule additional testing if you make significant changes to your service. You should meet with the IA team regularly to discuss ongoing changes.
-
+​
 A significant change could be when you:
-
+​
 * change a cloud service provider
-* change stored data, for example if you introduce new data which can be classified as [personally identifiable information (PII)](https://en.wikipedia.org/wiki/Personally_identifiable_information)
+* change stored data, for example if you introduce new data which can be classified as [Personally Identifiable Information (PII)](https://en.wikipedia.org/wiki/Personally_identifiable_information)
```

Comment:
instead of Personal Identifiable Data we now use Personal data as defined under GDPR https://ec.europa.eu/info/law/law-topic/data-protection/reform/what-personal-data_en